src/styles/Header.css

```diff
@@ -0,0 +1,3 @@
+.search-bar {
```

Comment:
When we have the styling files set up, this should probably be in our scss file of class names. I think we should also be using BEM

Reply:
I did this with the intention that I was doing it in BEM, but now I realize I wasn't. Nice catch. So would it be `.header__search`?